### DISCUSSION TITLE
How to hide the "Try NocoDB Cloud" Ads ?

### DISCUSSION THREAD

**Discussion by wzdxy** [nocodb/nocodb#8942] [Q&A]
<img width="290" alt="image" src="https://github.com/nocodb/nocodb/assets/12983764/16f3902b-867e-4549-b5ed-bc09779af849">
<img width="308" alt="image" src="https://github.com/nocodb/nocodb/assets/12983764/ee3800fd-3069-4be0-8148-005662d482f8">

Nocodb provides great spreadsheet functionality, and I deployed it on my own server. But this advertisement has nothing to do with the features provided by nocodb, and it will trouble my users.

Can I hide it ？

Thank you.

**Comment by derevyan** [comment]
I love nocodb but banner is annoying on self hosted version, i found a way to remove it - https://github.com/teableio/teable

**Reply by milgner** [reply]
Not really a solution. I came to nocodb from teable because it didn't allow me to add a second user for my buddy with whom I was trying to work without a commercial subscription.

**Comment by Anachron** [comment]
Yep, please allow us to hide this banner via an environment variable or alike.
